pkg/server/plugin/nodeattestor/k8s/sat/sat.go

```diff
@@ -229,6 +229,7 @@ func (p *AttestorPlugin) Attest(stream nodeattestorv1.NodeAttestor_AttestServer)
 					k8s.MakeSelectorValue("agent_ns", namespace),
 					k8s.MakeSelectorValue("agent_sa", serviceAccountName),
 				},
+				CanReattest: false,
```

Comment:
I wonder if we should mark SAT as true. If you re-attest using the same token, you end up with a different agent ID (suffixed with a random UUID). But from a "hey can we clean up this old agent record", the answer is yes. Does this align with your thinking as well, @evan2645 ?

Reply:
Makes sense, if the plugin can successfully reattest it should be set to true.